packages/server/src/index.ts

```diff
@@ -133,6 +133,16 @@ io.on("connection", (socket) => {
       await socket.leave(topic);
     }
   });
+
+  socket.on("select_seat", async ([game_id, seat, nr_of_players]) => {
```

Comment:
Nit: "select_seat" doesn't really communicate how this is different from "subscribe".  If it were me, I might call it something like "unique_subscribe" or something.

Also, do we any longer have a need for "subscribe"?  Should we deprecate it?

Reply:
I just checked, and [QuantumGo_Engine](https://github.com/SameerDalal/QuantumGo_Engine/blob/main/web_board.py#L162-L164) is not reliant on the socket directly, so no worries about breaking them either.